### PR TITLE
KFSPTS-23305 additional logging to figure out concur issue

### PR DESCRIPTION
**Pull request by jhulslander**
do NOT merge this PR.  Created PR just create a place to review this code.

We will rebuild the kfs-test WAR file with this feature branch, and then do Concur's "test connection". This additional logging should tell us what Concur is is putting in the HTTP request they are sending us.

The goal of this PR is to determine if another developer thinks additional logging may be helpful.

This code won't be merged to master, it is just being used as short term trouble shooting.

### CODE REVIEW

**Comment by jhulslander** [comment]
This PR is failing unit tests because the unit test jenkins job expect kew-to-kfs code, which this code doesn't have.

File: src/main/java/edu/cornell/kfs/concur/rest/authentication/ConcurAuthenticationFilter.java [hunk 1]
```diff
@@ -31,6 +33,7 @@ public void destroy() {
     public void doFilter(ServletRequest servletRequest, ServletResponse response, FilterChain filterChain) throws IOException, ServletException {
         LOG.debug("doFilter() started");
         HttpServletRequest httpServletRequest = (HttpServletRequest) servletRequest;
+        logRequst(httpServletRequest);
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/concur/rest/authentication/ConcurAuthenticationFilter.java:36]: spelling mistake, should be Request not Requst?

File: src/main/java/edu/cornell/kfs/concur/rest/authentication/ConcurAuthenticationFilter.java [hunk 2]
```diff
@@ -47,9 +50,28 @@ public void doFilter(ServletRequest servletRequest, ServletResponse response, Fi
                 return;
             }
         }
-        LOG.debug("doFilter(): Concur authentication was successful");
+        LOG.info("doFilter(): Concur authentication was successful");
         filterChain.doFilter(servletRequest, response);
     }
+    
+    private void logRequst(HttpServletRequest request) {
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/concur/rest/authentication/ConcurAuthenticationFilter.java:57]: code looks good, just has a spelling error in method name and where the method name is used in logging.

File: src/main/java/edu/cornell/kfs/concur/rest/authentication/ConcurAuthenticationFilter.java [hunk 3]
```diff
@@ -47,9 +50,28 @@ public void doFilter(ServletRequest servletRequest, ServletResponse response, Fi
...
+            }
+            
+            Enumeration<String> headerNames = request.getHeaderNames();
+            while (headerNames.hasMoreElements()) {
+                String headerKey = headerNames.nextElement();
+                String headerValue = request.getHeader(headerKey);
+                LOG.info("logRequst, headerKey: '" + headerKey + "' and header value: '" + headerValue + "'");
+            }
+        } else {
+            LOG.info("logRequst, this is production, so we aren't dooing this logging.");
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/concur/rest/authentication/ConcurAuthenticationFilter.java:72]: 'dooing' has spelling mistake.

**Review by db483 — CHANGES_REQUESTED** [review]
Changes look good, there are just some spelling mistakes.

**Comment by jhulslander** [comment]
Thanks, I fixed the spelling mistakes, I have a working session scheduled with Tami for Thursday

**Pull request closed without merging**